Reddit Wallpaper Changer/RWC.cs

```diff
@@ -1920,15 +1920,29 @@ private void AutoSave()
         {
             try
             {
-                if (!File.Exists(Properties.Settings.Default.defaultSaveLocation + @"\" + Properties.Settings.Default.currentWallpaperName))
+                String fileName = Properties.Settings.Default.currentWallpaperName;
+
+                // Remove illegal characters from the post title
+                bool changed = false;                    
+                foreach (char c in Path.GetInvalidFileNameChars())
                 {
+                    if (fileName.Contains(c))
+                        changed = true;                    
+                    fileName = fileName.Replace(c.ToString(), "");                                  
```

Comment:
/r/SubredditName is common here and becomes rSubredditName.
It would not bother me, but you could also replace that character with something else if you think its cleaner (maybe -r-SubredditName?)